06_Bulding_containers_from_conda_pip_environments/reference_solution/reference_solution.md

```diff
@@ -35,7 +37,7 @@ On a LUMI-C compute node, we may build the container non-interactively by:
 $ module purge
 $ module use /appl/local/training/modules/AI-20241126
 $ module load cotainr
-$ srun --output=cotainr.out --error=cotainr.err --account=project_465001363 --time=00:05:00 --mem=60G --cpus-per-task=8 --partition=dev-g cotainr build python312.sif --system=lumi-c --conda-env=python312.yml --accept-licenses
+$ srun --output=cotainr.out --error=cotainr.err --account=project_465001958 --time=00:15:00 --mem=100G --cpus-per-task=8 --partition=dev-g cotainr build python312.sif --base-image=/appl/local/containers/sif-images/lumi-rocm-rocm-6.0.3.sif --conda-env=examples/python312.yml --accept-licenses
```

Comment:
I would keep it to --mem=60G as this refers to 1 GPU equivalent (1/8th of one LUMI-G node). If you request 100G you will be billed for 2 GPUs